dotcom-rendering/src/web/components/Card/components/CardLink.tsx

```diff
@@ -39,3 +44,51 @@ export const CardLink = ({
 		/>
 	);
 };
+
+const ExternalLink = ({
+	linkTo,
+	headlineText,
+	dataLinkName,
+}: {
+	linkTo: string;
+	headlineText: string;
+	dataLinkName?: string;
+}) => {
+	return (
+		// eslint-disable-next-line jsx-a11y/anchor-has-content -- we have an aria-label attribute describing the content
+		<a
+			href={linkTo}
+			css={fauxLinkStyles}
+			data-link-name={dataLinkName}
+			aria-label={headlineText + ' (opens in new tab)'}
+			target="_blank"
+			rel="noreferrer"
+		/>
+	);
+};
+
+export const CardLink = ({
+	linkTo,
+	headlineText,
+	dataLinkName = 'article', //this makes sense if the link is to an article, but should this say something like "external" if it's an external link? are there any other uses/alternatives?
```

Comment:
this is a good question! seems like it's probably worth following up with data team?